src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,27 +1,40 @@
 package com.kodilla.ecommercee.controller;
 
 import com.kodilla.ecommercee.domain.dto.UserDto;
-import org.springframework.web.bind.annotation.*;
```

Comment:
Patrząc na wcześniejsze review, mieliśmy unikać importowania poprzez "*" tylko wskazywać konkretne importy.

Reply:
Słusznie, poprawię zaraz, musiałem przestawić intellij bo mi to zmieniał cały czas.